lib/salus/scanners/report_ruby_gems.rb

```diff
@@ -26,8 +26,11 @@ def should_run?
 
     def record_dependencies_from_gemfile_lock
       lockfile = Bundler::LockfileParser.new(@repository.gemfile_lock)
-      record_ruby_version(lockfile.ruby_version, 'Gemfile')
-      record_bundler_version(lockfile.bundler_version, 'Gemfile')
+
+      # N.B. lockfile.bundler_version is a Gem::Version
```

Comment:
what does this note mean?

Reply:
`lockfile.bundler_version` isn't a string (it's a Gem::Version), which could cause some surprising breakage if you didn't stringify it first